cf_remote/commands.py

```diff
@@ -403,6 +403,9 @@ def spawn(
         role=role,
         spawned_cb=print_progress_dot,
     )
+    if len(vms) == 0:
```

Comment:
You could do
```suggestion
    if vms:
```
since empty list evaluates to `False`, but maybe your way is more readable 🤔

Reply:
```
>>> if not []:
...     print("Empty")
... else:
...     print("Not empty")
... 
Empty
```

Reply:
I think maybe your way is clearer 😉

Reply:
:blush:

Reply:
You've updated the function to return None, this won't work;

```
$ python3
Python 3.11.6 (main, Oct  2 2023, 20:14:46) [Clang 14.0.0 (clang-1400.0.29.202)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> len(None)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: object of type 'NoneType' has no len()
```